### PR TITLE
Add shellcheck image version 0.7.1

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+v0.7.1
```
Comment (ericwill) [inline VERSION]: New line missing at EOF

**Review by ericwill — COMMENTED** [review]
Looks like are `arm64` is available for this image as well, may as well add it in. All you need to do is add `linux/arm64` to the PLATFORMS file, like [this](https://github.com/che-dockerfiles/che-sidecar-node/blob/master/PLATFORMS).

File: VERSION (hunk 1 above)
Comment (benoitf) [reply VERSION]: I fixed the EOF

**Comment by benoitf** [comment]
I've added `linux/arm64` in `PLATFORMS` file

**Review by benoitf — APPROVED** [review]

**Pull request merged**